examples/notebooks/persistent_dataset_speed.ipynb

```diff
@@ -0,0 +1,6871 @@
+    "train_labels = glob.glob(os.path.join(data_root, 'labelsTr', '*.nii.gz'))\n",
+    "data_dicts = [{'image': image_name, 'label': label_name}\n",
+    "              for image_name, label_name in zip(train_images, train_labels)]\n",
+    "train_files, val_files = data_dicts[:-9], data_dicts[-9:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup transforms for training and validation\n",
+    "  \n",
+    "Deterministic transforms during training:  \n",
+    "- LoadNiftid  \n",
+    "- AddChanneld  \n",
+    "- Spacingd  \n",
+    "- Orientationd  \n",
+    "- ScaleIntensityRanged  \n",
+    "  \n",
+    "Non-deterministic transforms:  \n",
+    "- RandCropByPosNegLabeld  \n",
+    "- ToTensord\n",
+    "\n",
+    "All the validation transforms are deterministic.  \n",
+    "The results of all the deterministic transforms will be cached to accelerate training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_transforms = Compose([\n",
+    "    LoadNiftid(keys=['image', 'label']),\n",
+    "    AddChanneld(keys=['image', 'label']),\n",
+    "    Spacingd(keys=['image', 'label'], pixdim=(1.5, 1.5, 2.), interp_order=(3, 0)),\n",
+    "    Orientationd(keys=['image', 'label'], axcodes='RAS'),\n",
+    "    ScaleIntensityRanged(keys=['image'], a_min=-57, a_max=164, b_min=0.0, b_max=1.0, clip=True),\n",
+    "    # randomly crop out patch samples from big image based on pos / neg ratio\n",
+    "    # the image centers of negative samples must be in valid image area\n",
+    "    RandCropByPosNegLabeld(keys=['image', 'label'], label_key='label', size=(96, 96, 96), pos=1,\n",
+    "                           neg=1, num_samples=4, image_key='image', image_threshold=0),\n",
+    "    ToTensord(keys=['image', 'label'])\n",
+    "])\n",
+    "\n",
+    "# NOTE: No random cropping in the validation data, we will evaluate the entire image using a sliding window.\n",
+    "val_transforms = Compose([\n",
+    "    LoadNiftid(keys=['image', 'label']),\n",
+    "    AddChanneld(keys=['image', 'label']),\n",
+    "    Spacingd(keys=['image', 'label'], pixdim=(1.5, 1.5, 2.), interp_order=(3, 0)),\n",
+    "    Orientationd(keys=['image', 'label'], axcodes='RAS'),\n",
+    "    ScaleIntensityRanged(keys=['image'], a_min=-57, a_max=164, b_min=0.0, b_max=1.0, clip=True),\n",
+    "    ToTensord(keys=['image', 'label'])\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `Dataset`: Define, train and validate\n",
+    "\n",
+    "Load each original dataset and transform each time it is needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Using 16 cores for preprocessing\n",
```

Comment:
Let's remove the training outputs(too long) and only keep the plot charts?
Thanks.